native/ios/native/Info.plist

```diff
@@ -41,6 +41,7 @@
 	<array>
 		<string>BpkIcon.ttf</string>
 		<string>BpkIcon.woff</string>
+		<string>BpkIconIOS.ttf</string>
```

Comment:
We generate `eot`, `ttf` and `woff` but only add the `ttf` in the Xcode project files? For comparison, the existing `BpkIcon` is included as `ttf` and `woff` ...

Reply:
Dunno, don't care